src/main/resources/templates/admin/logistic/index.html

```
@@ -0,0 +1,167 @@
+<!DOCTYPE html>
+<html lang="pt-br" xmlns:th="http://www.thymeleaf.org">
+  <head>
+    <div th:replace="~{fragments/head_info.html :: head_info}"></div>
+    <link rel="stylesheet" type="text/css" th:href="@{/css/profile.css}" />
+    <link rel="stylesheet" type="text/css" th:href="@{/css/navbar.css}" />
+    <script src="/js/logistic.js" defer></script>
+    <script src="/js/bibliotecas/alpine.js" defer></script>
+  </head>
```

HTMLHint (1.5.0)

[error] <title> must be present in <head> tag.

(title-require)